net/portmapper/portmapper.go

```diff
@@ -808,7 +808,7 @@ func (c *Client) Probe(ctx context.Context) (res ProbeResult, err error) {
 					c.mu.Unlock()
 					continue
 				}
-				c.logf("unexpected PMP probe response: %+v", pres)
+				c.logf("PMP probe failed: %+v", pres)
```

Comment:
Line 801 lists the type we're expecting.

If there are more types we're expecting, I'd prefer to add new lines to recognize other things (even if we do nothing and log nothing).

But I'd like to keep this line for learning about even weirder things.

Reply:
I mean, this is also fine as-is (it's just rewording it), but maybe we want to recognize code 3 or whatever and log a specific message for that?

Reply:
Done. I kept all the detail, in case we want to do analytics about why NAT-PMP gets rejected.